### DISCUSSION TITLE
List format bulletpoints/dots is broken on Firefox browsers, works on Chromium browsers

### DISCUSSION THREAD

**Discussion by xXBuilderBXx** [GitbookIO/community#643] [0. General bugs and questions]
Looks like the list format with bulletpoints/dots is broken on Firefox.

Here's what it't showing
![image](https://github.com/GitbookIO/community/assets/17956143/8aed3e41-6800-49fa-9c3a-447615fff4af)

Here's what it should be showing (chromium browsers)
![image](https://github.com/GitbookIO/community/assets/17956143/f5781681-c33f-4273-bfc3-d16c97059de7)

This makes the docs looks a lot cluttered and harder to read especially if the bulletpoints are wordwrapped or have multiple lines you don't know where the start or end is.

**Comment by john-gitbook** [comment]
hey @xXBuilderBXx thanks for flagging this! 

I haven't been able to replicate this so I'll need to see your space to debug. Could you send us a message on our support chat [here](https://www.gitbook.com/contact) and include a link to your docs site? I can then investigate the issue for you.
